x/datapool/keeper/pool_distribution.go

```diff
@@ -163,6 +163,33 @@ func (k Keeper) sendDepositToCurator(ctx sdk.Context, pool *types.Pool) error {
 	return nil
 }
 
+func (k Keeper) executeRevenueDistribution(ctx sdk.Context, pool *types.Pool) error {
+	availablePoolCoinAmount, err := k.getAvailablePoolCoinAmount(ctx, pool)
+	if err != nil {
+		return err
+	}
+
+	// if there is no coin available, proceed no further.
+	if availablePoolCoinAmount.Equal(sdk.NewInt(0)) {
+		return nil
+	}
+
+	// calculate the revenue to be sent to each seller and curator
+	eachSellerDistributionAmount, curatorCommissionAmount := k.getEachDistributionAmount(pool)
```

Comment:
For clearer expression, I suggest to rename the `eachSellerDistributionAmount` to `eachDataDistributionAmount` or `distributionAmountPerData`, because it is the amount to be distributed per data.

Reply:
Oh right.
This is something we talked about earlier in the documentation.
I will change to `distributionAmountPerData`